### ISSUE TITLE
Add Tuner Number, Name and Tuner Type to /web/tunerinfo

### ISSUE THREAD

**Issue by phunkyfish** [E2OpenPlugins/e2openplugin-OpenWebif#854]
Would it be possible add tuner number, name and tuner type to the above call?

Example: 
Tuner name=Tuner I
Tuner Type=DVB-S2

`
<e2frontendstatus>
	<e2snrdb>84 dB</e2snrdb>
	<e2snr>84 %</e2snr>
	<e2ber>0</e2ber>
	<e2acg>66 %</e2acg>
	<e2tunername>Tuner I</e2tunername>
	<e2tunernumber>0</e2tunernumber>
	<e2tunertype>DVB-S2</e2tunertype>
</e2frontendstatus>
`

**Comment by jbleyel** [comment]
Tuner info is already there ../deviceinfo.
Why do you need another api?

**Comment by phunkyfish** [comment]
You're right I don't and in fact /deviceinfo would not provide what I need either.

The kodi addon streams video from the STB. In actuality what I would need is the tuner info for a particular stream. Would that even be possible?

So an api passing the stream I am using, which then returns the tunerinfo for the tuner in use. The tuner the STB is displaying is not relevant.

Interested to hear your thoughts?

**Comment by phunkyfish** [comment]
What is the difference between live stream and live playback?

**Comment by jbleyel** [comment]
Live playback means output via HDMI.
Live stream means network stream.
Recording means save output as file.

All 3 types uses tuner.
The current used tuner info will not be available in all of these cases.

**Comment by phunkyfish** [comment]
So my case would be live stream only, it’s all over the network.

Is it possible to provide tuner info in this case? What arguments would you potentially need to be able to return this?

**Comment by phunkyfish** [comment]
If the /web/tunersignal API could take an optional argument of tuner name then if that tuner is found it could return the signal info for that tuner, error otherwise.

In addition to this a new API would be needed that lists the live streams and the tuners they are running on. It could also be incorporated into an existing API such deviceinfo that already lists the tuners.
